### ISSUE TITLE
[Accessibility] Link address are cut in chronological tabs

### ISSUE THREAD

**Issue by SimonBasca** [mozilla-mobile/firefox-ios#8527]
## Steps to reproduce
1. Increase large text accessibility 
2. Visit a few pages
3. Open chronological tabs
### Expected behavior
-  Link address should be visible in chronological tabs
### Actual behavior
- Link address are cut in chronological tabs
### Device & build information
* Device: iPhone Xr (14.4)
* Build version: 34.0 (4847)
### Notes
Attachments: 
![Tabs Tray](https://user-images.githubusercontent.com/32130829/118645110-3174d200-b7e7-11eb-8e00-8bb372c715df.jpeg)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-2060)
